loops-and-arrays.js

```diff
@@ -57,5 +71,8 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 11 is 1331'
 // ]
 
+for(let i=0 ; i<squares.length;i++){
+    squares[i] = "Het volume van " + squares[i] + " is " + squares[i]**3;
```

Comment:
Zorg ervoor dat de haakjes correct zijn gesloten. In de huidige code ontbreekt een haakje aan het einde van de regel:
squares[i] = "Het volume van " + squares[i] + " is " + squares[i]**3;